### PR TITLE
Fix links to /blog

### PR DESCRIPTION
**Pull request by w0st**
Reason: `.Site.BaseURL` contains the trailing slash. It could redirect the user to `https://blog/` in the browser.

### CODE REVIEW

File: themes/fromAtoB/layouts/index.html [hunk 1]
```diff
@@ -122,7 +122,7 @@ <h2 class="section-heading">Blog</h2>
             <p>
               We maintain an engineering blog where we post some cool stuff.
             </p>
-            <a href="{{ .Site.BaseURL }}/blog" class="btn btn-primary btn-xl sr-button">Check out Eng Blog</a>
+            <a href="{{ .Site.BaseURL }}blog" class="btn btn-primary btn-xl sr-button">Check out Eng Blog</a>
```
Comment (lucasrenan) [inline themes/fromAtoB/layouts/index.html]: I was expecting `.Site.BaseURL` to return: https://github.com/fromAtoB/blog/blob/master/config.toml#L3 🤔

Comment (w0st) [reply themes/fromAtoB/layouts/index.html]: Right :man_facepalming: I'll update it.

Comment (lucasrenan) [reply themes/fromAtoB/layouts/index.html]: maybe we should be using `absURL` instead https://gohugo.io/functions/absurl/
WDYT?

Comment (w0st) [reply themes/fromAtoB/layouts/index.html]: I've checked. It works as you expected if you check the generated files under `/public`. The links lead to absolute path: `https://eng.fromatob.com/blog`
According to https://gohugo.io/getting-started/quick-start/#site-configuration it seems that the baseurl isn't considered during development mode (`hugo server`)

**Pull request merged**